test_vm/vendor/cookbooks/dbfit_test/recipes/default.rb

```diff
@@ -1,6 +1,8 @@
 include_recipe "dbfit_test::diskspace"
-include_recipe "dbfit_test::firewalld"
+include_recipe 'firewalld::disable'
```

Comment:
Hmm. Is that firewall change necessary?
If we want to disable firewall - maybe we should also remove dbfit `firewalld.rb` recipe. Or alternatively we can tweak `firewalld.rb`, open more ports, etc. instead of disabling it.